lua/smh/client/derma/settings.lua

```diff
@@ -42,7 +46,7 @@ local function Create(parent)
 	panel.HelpButton = vgui.Create("DButton", panel);
 	panel.HelpButton:SetText("Help");
 
-	panel:SetSize(160, 225);
+	panel:SetSize(160, 245); --225
```

Comment:
No need to keep the original value as comment; git history exists for that reason :)

Reply:
Yeah, I've mostly left the comment there for myself and forgot to remove it